log-manager/src/main/java/io/airlift/log/LoggingConfiguration.java

```diff
@@ -140,7 +140,11 @@ public String getLogAnnotationFile()
         return logAnnotationFile;
     }
 
-    @Config("log.annotation-file")
+    /* Reuse the same property as defined in io.airlift.node.NodeConfig because NodeInfo objects
```

Comment:
Nit: we would use `//` comments for this. Using a block comment here makes it look like a Javadoc.

Reply:
Thanks, it felt weird but I wasn't sure what's the convention.